app/src/main/kotlin/com/wire/android/navigation/NavigationUtils.kt

```diff
@@ -34,3 +36,17 @@ internal fun NavController.getCurrentNavigationItem(): NavigationItem? {
     val currentRoute = navBackStackEntry?.destination?.route
     return NavigationItem.fromRoute(currentRoute)
 }
+
+internal fun QualifiedID.mapIntoArgumentString(): String = "$domain@$value"
```

Comment:
I think we already have an extension function for that `.asString()`  in `QualifiedID` file...If not could you move it there...Mapping is not something related to navigation only

Reply:
I moved it to `QualifiedId` class file